contracts/scripts/ousd-governance/deployContracts.js

```diff
@@ -0,0 +1,153 @@
+const fs = require("fs");
```

Comment:
Any reason why not put these things under `contracts/deploy` and make it utilise [hardhat deploy](https://www.npmjs.com/package/hardhat-deploy)

Reply:
Going forward I think it would be a great benefit to using a unified way to deploy contracts. You have probably given this more though and might know of some limitation I don't ofc. So do tell please :)

Reply:
I agree on this, integrated the governance deployment scripts into the 001_core deployment